asciidoctorj-core/src/main/java/org/asciidoctor/extension/Contexts.java

```diff
@@ -1,10 +1,6 @@
 package org.asciidoctor.extension;
 
-import java.lang.annotation.Documented;
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.annotation.Target;
+import java.lang.annotation.*;
```

Comment:
Nitpick: I generally never use star imports. I'm fine with it, but want to make sure this didn't slip in accidentally, because the default of IntelliJ seems to use them starting with 4 imported elements.

Reply:
Indeed it's the default behavior of IntelliJ but I can revert this change :wink:

Reply:
I'm fine with it. :) I just wanted to be certain that we want this.
If you think it makes sense to have star imports I'll merge.

(Of course I have to admit that there are parts of the code where the formatting is not glorious 😄)

Reply:
I can find it anymore but Mark Reinhold tweeted something like "you should use this language feature" about star imports but at the same time I think it's better to be explicit.
I don't have a strong opinion so explicit imports is fine (and it also consistent with the codebase).